macchiato.go

```diff
@@ -5,7 +5,7 @@ type macchiato struct{}
 
 var _ Theme = macchiato{}
 
-func (macchiato) Name() string { return "macchiato" }
+func (macchiato) Name() string { return "Macchiato" }
```

Comment:
Same comment as above.
```suggestion
func (macchiato) Name() string { return "macchiato" }
```